### PR TITLE
Claw tests

### PR DESCRIPTION
**Pull request by Drfeelgood999**
Before issuing a pull request, please see the contributing page.

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/ElevatorMove.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/ElevatorMove.java:1]: Why two Commands and Command Folder?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/DashboardUtil.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/DashboardUtil.java:1]: Make sure that the Utils Folder exists but it's empty for the JoystickHandler.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Encoder.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Encoder.java:1]: Make sure that the files from the roadrunner library are all deleted.

**Review by SantiQ0905 — CHANGES_REQUESTED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/DashboardUtil.java
Comment (CristobalCastilloEnriotti) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/DashboardUtil.java:1]: 👍

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/ElevatorMove.java
Comment (Drfeelgood999) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/ElevatorMove.java:1]: fixed...!

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/DashboardUtil.java
Comment (Drfeelgood999) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/DashboardUtil.java:1]: fixed

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Encoder.java
Comment (Drfeelgood999) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Encoder.java:1]: fixeddd

**Review by SantiQ0905 — APPROVED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ElevatorMove.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import org.firstinspires.ftc.teamcode.Subsystems.Elevator;
+
+public class ElevatorMove extends CommandBase {
+
+    private Elevator elevator;
+    private double voltage;
+
+    //Este double va a ser parte del calculo para saber que tanto se tiene que extender
+    private double elevatorTimeToExtend;
+    private final double extendedHeight = 15; //Valor no final
```
Comment (AJahueyM) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ElevatorMove.java]: Remove unused variables

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ElevatorMove.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    public ElevatorMove(Elevator subsystem, double elevatorHeight) {
+        this.elevatorHeight = elevatorHeight;
+        elevator = subsystem;
+        addRequirements(subsystem);
+    }
+
+    @Override
+    public void initialize() {
+        elevator.ElevatorHeight(elevatorHeight);
+        elevator.ElevatorHeight(elevatorHeight);
```
Comment (AJahueyM) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ElevatorMove.java]: Why the double call to elevator.ElevatorHeight?

**Review by AJahueyM — CHANGES_REQUESTED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java [hunk 3]
```diff
@@ -61,15 +93,19 @@ public void runOpMode() {
 
         waitForStart();
 
+
         while (opModeIsActive()) {
             CommandScheduler.getInstance().run();
 
+        waitForStart();
+
+        while (opModeIsActive()) {
```
Comment (AJahueyM) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java]: but why

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Claw.java [hunk 4]
```diff
@@ -1,22 +1,29 @@
...
+    private Servo twist_Left; //Declare left arm
 
     public Claw (HardwareMap hardwareMap) {
         // Servos IDs
         grab_RightServo = hardwareMap.get(Servo.class, "grab_RightServo");
         grab_LeftServo = hardwareMap.get(Servo.class, "grab_LeftServo");
+        twist_Right = hardwareMap.get(Servo.class, "twist_Right");
+        twist_Left = hardwareMap.get(Servo.class, "twist_Left");
         // Reverse the reversed Servo
         grab_RightServo.setDirection(Servo.Direction.FORWARD);
```
Comment (AJahueyM) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Claw.java:23]: but why

**Review by AJahueyM — CHANGES_REQUESTED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java (hunk 3 above)
Comment (SantiQ0905) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java]: beacuse reasons and life sometimes is cruel im sorry alberto wont happen again i love u

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Claw.java (hunk 4 above)
Comment (SantiQ0905) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Claw.java:23]: im sorry but here is correct no problem theres 4 servos left rigth twist left right grab

**Review by SantiQ0905 — APPROVED** [review]

**Pull request merged**